### DISCUSSION TITLE
Creating a pinned READme for users.

### DISCUSSION THREAD

**Discussion by Anyaoha** [gochain-io/docs#146] [Show and tell]
I have created a READme that will be pasted on top of the the GoChain repo here for users to easily find out what't going on with the project rather than tediously going through all the repos. 

I need someone within the org that has a write access to create a new ".github" repo with a profile path that contains a README.md file so I can create the doc.

**Comment by treeder** [comment]
Hi, can you share the README so we can see what you have in mind. Thanks!

**Reply by Anyaoha** [reply]
sure take a look at the overview profile highlights I've made for GoChain:
 https://github.com/Anyaoha/GoChain-profile.

Let me know what you think

**Reply by treeder** [reply]
Created and left it mostly blank so you can create a pull request and take credit for it if you'd like. https://github.com/gochain/.github/tree/main/profile
